app/views/catalog/facet.html.erb

```diff
@@ -6,6 +6,7 @@
   <% end %>
 
   <% component.with_title { facet_field_label(@facet.key) } %>
+  <%= render Blacklight::Search::FacetSuggestInput.new(facet: @facet, presenter: @presenter) %>
```

Comment:
Should we make this opt-in with facet configuration in order to keep this backwards compatible?

Reply:
Added as a configuration option, great idea, thanks!